src/Fixer/Casing/ConstantCaseFixer.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Casing;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * Fixer for rules defined in PSR2 ¶2.5.
+ *
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+final class ConstantCaseFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
+{
+    /**
+     * Hold the function that will be used to convert the constants.
+     *
+     * @var callable
+     */
+    private $fixFunction;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        parent::configure($configuration);
+
+        if ('lower' === $this->configuration['case']) {
+            $this->fixFunction = 'strtolower';
+        }
+
+        if ('upper' === $this->configuration['case']) {
```

Comment:
`else` instead of this `if`?

Reply:
I'm always trying to avoid `else|elseif` conditions to have a easier code to read and to usually reduce the cyclomatic complexity. Is this really necessary ?